### PR TITLE
CURA-4211 New extruder buttons

### PR DESCRIPTION
**Pull request by LipuFei**
image

### CODE REVIEW

**Comment by LipuFei** [comment]
Fixed the merge conflicts and some other things. This is the new look:
new-1
new-2

File: resources/qml/ExtruderButton.qml [hunk 1]
```diff
@@ -54,22 +49,27 @@ Button
...
         color: model.color
 
-        width: UM.Theme.getSize("default_margin").width
-        height: UM.Theme.getSize("default_margin").height
+        width: UM.Theme.getSize("extruder_button_material_size").width
+        height: UM.Theme.getSize("extruder_button_material_size").height
+        radius: width / 2
+
+        border.width: 0
+        border.color: "transparent"
```
Comment (nallath) [inline resources/qml/ExtruderButton.qml]: Do we really need to set color if the border if it's already at 0 size?

Comment (LipuFei) [reply resources/qml/ExtruderButton.qml]: That can be gone.

**Comment by maukcc** [comment]
Maybe also color the entire circel in which the number is when the extruder is actually used?

**Comment by fieldOfView** [comment]
First of all, thanks for also testing this with 8 extruders.
Nitpick: there's a double border between extruder buttons that looks ugly.

**Pull request merged**